AppInspector.CLI/Writers/VerifyRulesJsonWriter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.ApplicationInspector.Commands;
+using Newtonsoft.Json;
+
+namespace Microsoft.ApplicationInspector.CLI.Writers
+{
+    class VerifyRulesJsonWriter : CommandResultsWriter
+    {
+        public override void WriteResults(Result result, CLICommandOptions commandOptions, bool autoClose = true)
+        {
+            WriteOnce.Result("Result");
+
+            JsonSerializer jsonSerializer = new JsonSerializer();
+            jsonSerializer.Formatting = Formatting.Indented;
+            jsonSerializer.Serialize(TextWriter, (VerifyRulesResult)result);
+
+            WriteOnce.NewLine();
+
+            if (autoClose)
+                FlushAndClose();
+        }
+
+        public override void FlushAndClose()
+        {
+            TextWriter.Flush();
+            TextWriter.Close();
```

Comment:
You're not setting WriteOnce.TextWriter to null here but you are in other TextWriters

Reply:
good catch.

Reply:
fixed